app/components/post/post_container.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+
+import {getTheme} from 'service/selectors/entities/preferences';
+import {getUser} from 'service/selectors/entities/users';
+
+import Post from './post';
+
+function mapStateToProps(state, ownProps) {
+    return {
+        ...ownProps,
+        user: getUser(state, ownProps.post.user_id),
+        post: ownProps.post,
```

Comment:
also I think this props should be passed from the **renderPost** method in `PostList` as this should be a dumb component?